### PR TITLE
Specify correct tags and build age for Veneur check

### PR DESCRIPTION
**Pull request by ChimeraCoder**
This makes our graphs show up right

r? @cory-stripe || @joshu-stripe 

cc @stripe/observability

### CODE REVIEW

File: checks.d/veneur.py [hunk 1]
```diff
@@ -25,11 +25,13 @@ def check(self, instance):
             builddate = datetime.datetime.fromtimestamp(int(r.text))
 
             tdelta = datetime.datetime.now() - builddate
-            self.histogram(self.BUILDAGE_METRIC_NAME, tdelta.seconds)
```
Comment (cory-stripe) [inline checks.d/veneur.py]: Does this also need the `instance`'s tags?

Comment (ChimeraCoder) [reply checks.d/veneur.py]: I think those get set by default? Looking at `/metric/summary?filter=build_age`.

Comment (cory-stripe) [reply checks.d/veneur.py]: You applied them to the gauge below though?

Comment (ChimeraCoder) [reply checks.d/veneur.py]: Because otherwise it seems we overwrite them when passing a new value for `tags`. The default is safe, though.

**Comment by cory-stripe** [comment]
👍 

Oh, ugh, sorry, i missed the mutation of the tags.

**Pull request merged**